### PR TITLE
Fix: Dismiss keyboard on opening navigation drawer

### LINKED ISSUE #1234: Keyboard related issue
**Summary:** 

On the Dashboard page keyboard is not closing when the navigation drawer is opened. This affects the user experience.

**Steps to reproduce:** 

Just open the keyboard by clicking on search box then open navigation drawer.

**Expected behaviour:** 

The keyboard must be closed while opening the navigation drawer.

**Observed behaviour:** 

On the Dashboard page keyboard is not closing when the navigation drawer is opened.

**Device and Android version:** 

Realme 5
Android version 9.0

**Screenshots:** 

issue13

### PR DESCRIPTION
**Pull request by iamsh4shank**
Fixes #1234 

Please Add Screenshots If there are any UI changes.
resolved1

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `MifosStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by robustTechie** [comment]
Please review my pull request.

File: mifosng-android/src/main/java/com/mifos/mifosxdroid/core/MifosBaseActivity.java [hunk 1]
```diff
@@ -121,10 +121,9 @@ public void hideProgress() {
     }
 
     public void hideKeyboard(View view) {
-        InputMethodManager inputManager = (InputMethodManager) this
-                 .getSystemService(Context.INPUT_METHOD_SERVICE);
-        inputManager.hideSoftInputFromWindow(getWindow().getAttributes().token, InputMethodManager
-                 .RESULT_UNCHANGED_SHOWN);
+        InputMethodManager inputMethodManager = (InputMethodManager)
+                getSystemService(Context.INPUT_METHOD_SERVICE);
+        inputMethodManager.hideSoftInputFromWindow(getCurrentFocus().getWindowToken(), 0);
```
Comment (laxyapahuja) [inline mifosng-android/src/main/java/com/mifos/mifosxdroid/core/MifosBaseActivity.java]: Hey @robustTechie. Using this line of code causes [this](https://github.com/openMF/android-client/issues/1219) issue. I had fixed it in [this](https://github.com/openMF/android-client/issues/1219) PR. What you're doing right now will cause the issue to occur again. Leaving this block of code as it was will work the same way, I guess.

Comment (robustTechie) [reply mifosng-android/src/main/java/com/mifos/mifosxdroid/core/MifosBaseActivity.java]: After adding this code the keyboard is dismissed if I open the navigation drawer without causing any application crash.

**Comment by iamsh4shank** [comment]
Already updated in the app in another PR

**Pull request closed without merging**